virtual_ecosystem/models/soil/all_pools.py

```diff
@@ -1,15 +1,16 @@
-"""The ``models.soil.carbon`` module  simulates the soil carbon cycle for the Virtual
-Ecosystem. At the moment five pools are modelled, these are low molecular weight carbon
+"""The ``models.soil.all_pools`` module simulates all soil pools for the Virtual
+Ecosystem. At the moment four carbon pools are modelled (low molecular weight carbon
 (LMWC), mineral associated organic matter (MAOM), microbial biomass, particulate organic
-matter (POM), and POM degrading enzymes.
+matter (POM)), as well as two enzyme pools (POM and MAOM) degrading enzymes, and two
+nitrogen pools (dissolved organic nitrogen and particulate organic nitrogen).
```

Comment:
Add their acronyms as well, for consistency with other pools.

```suggestion
nitrogen pools (dissolved organic nitrogen (DON) and particulate organic nitrogen).
```